api/pending_test_runs.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019 The WPT Dashboard Project. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package api
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/web-platform-tests/wpt.fyi/shared"
+)
+
+// apiPendingTestRunsHandler is responsible for emitting JSON for
+// all the pending test runs.
+func apiPendingTestRunsHandler(w http.ResponseWriter, r *http.Request) {
+	ctx := shared.NewAppEngineContext(r)
+	store := shared.NewAppEngineDatastore(ctx, false)
+
+	q := store.NewQuery("PendingTestRun").Order("-Updated")
+	var runs []shared.PendingTestRun
+	if keys, err := store.GetAll(q, &runs); err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	} else {
+		for i, key := range keys {
```

Comment:
Sigh, golint doesn't like it:
/home/user/go/src/github.com/web-platform-tests/wpt.fyi/api/pending_test_runs.go:25:9: if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary)